schema/vrs.yaml

```diff
@@ -678,3 +710,117 @@ definitions:
+      # alternate representation of multiple breaks and we want to minimise the number
+      # of different ways a sequence can be represented.
+      sequence:
+        type: LiteralSequenceExpression
+        description:
+          # TODO: clarify what this sequence is. We can define this as:
+          # - Traversal from the anchoring sequence (i.e RevComp DivergesBefore sequences)
+          # - Sequence prepend/concatenation
+          # TODO: What happens when the sequence itself has DerivedSequenceExpression.reverse_complement=true?
+          Sequence occuring after the break.
+      terminal:
+        # TODO: can the schema encode a constraint that a terminal breakend cannot
+        # be part of a breakpoint?
+        type: boolean
+        default: false
+        description: Indicates the end of the molecule
+    required:
+      - type
+      - location
+      - orientation
+    additionalProperties: false
+  Breakpoint:
+    description:
+      A rearrangement resulting in sequences flanking the two breakends becoming
+      adjacent sequences on the same molecule.
+    type: object
+    properties:
+      type:
+        type: string
+        const: Breakpoint
+        default: Breakpoint
+        description: MUST be "Breakpoint"
+      # I've taken the VCF-style model
+      # This is actually a lossy representation as many variant callers
+      # can constrain the actual location much more than anywhere in the
+      # [(start1, end1), (start2, end2)] range.
+      # for example, when the interval are due to homology, then the
+      # interval widths must be the same and, for any given position
+      # in the first breakend interval, there is only one possible position
+      # in the second breakend interval that is possible.
+      # Just specifying the two intervals independently as is done in this
+      # model does not intrinsically encode this information.
+      #
+      # Similarly, even imprecise calls have possibilities that are less
+      # likely than others. For example if a deletion break1 was at start1, then
+      # break2 might be constrained so something like [start2, start2 + (end2 - start2) / 3]
+      # because that would imply a longer deletion length that is plausible.
+      #
+      # VCF has a CILEN field that can encode this sort of information.
+      # Do we need an equivalent for VRS?
+      breakends:
+        type: array
+        uniqueItems: false
+        ordered: false
+        items:
+          type: Breakend
+        # TODO: how should the Breakend.sequence be interpreted?
+        # Option 1: be1 <-> seq1 <-> seq2 <-> be2
+        # That is, concatenate the sequences of both of them.
+        # Pros: no possibility of data inconsistency
+        # Cons: need to 'allocate' the sequence to one of the breakends
+        #
+        # Option2: be1 <-> seq <-> be2
+        # Pros: more intuitive interpretation
+        # Cons: possibility of data inconsistency (when seq1 != seq2)
```

Comment:
How would `seq` be defined in option 2?

Reply:
From a discussion with Daniel, the reason for putting `sequence` at the breakend-level is that we also need to support inserted sequence on terminal breakends.

Another possibility is representing terminal breakends as a breakpoint with a single terminal breakend, and moving `sequence` up to the breakpoint.